rcongui/Dockerfile

```diff
@@ -2,26 +2,30 @@ FROM node:13-stretch
 
 WORKDIR /code
 
-RUN apt-get update -y && apt-get install -y nginx apache2-utils
+RUN apt-get update -y && apt-get install -y yarn nginx apache2-utils
 COPY nginx.conf /etc/nginx/sites-available/default
 
 RUN mkdir /pw
 
 COPY package.json .
-COPY package-lock.json .
+COPY yarn.lock .
 
-RUN npm install
+RUN yarn install
 
 ENV REACT_APP_API_URL /api/
 
 COPY src/ src/
 COPY public/ public/
-
-RUN npm run build
-
 COPY entrypoint.sh .
+
 RUN chmod +x /code/entrypoint.sh
 
+RUN yarn run build
+
+RUN rm -rf src/
+RUN rm -rf public/
```

Comment:
Are you sure static assets are in the build then?

Reply:
![Capture](https://user-images.githubusercontent.com/8644236/90763085-6569a800-e2e6-11ea-870f-75a7934cd104.JPG)
Yes public directory is copied into build one.